Formula/n98-magerun.rb

```diff
@@ -0,0 +1,17 @@
+class N98Magerun < Formula
+  desc "Swiss army knife for Magento developers, sysadmins and devops."
+  homepage "http://magerun.net/"
+  url "https://files.magerun.net/n98-magerun.phar"
```

Comment:
This needs to be a URL for a specific version e.g. https://files.magerun.net/n98-magerun-1.93.0.phar